packages/hdwallet-seeker/src/adapter.ts

```
@@ -0,0 +1,210 @@
+    return Promise.resolve('1.0.0')
+  }
+
+  getModel(): Promise<string> {
+    return Promise.resolve('Seeker')
+  }
+
+  getLabel(): Promise<string> {
+    return Promise.resolve('Seeker Wallet')
+  }
+
+  getPublicKeys(_msg: GetPublicKey[]): Promise<(PublicKey | null)[] | null> {
```

ESLint

[error] '_msg' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] Insert `;`

(prettier/prettier)